### PR TITLE
feat(FieldSelectMulti): Google Material label style

### PR DESCRIPTION
**Pull request by mdodgelooker**
Use `FloatingLabelField` in `FieldSelectMulti`.

I saved this for its own PR because I somehow thought I would have to deprecate `defaultValues`, but I was wrong. 🙂

### CODE REVIEW

**Comment by laragoogle** [comment]
when out of focus the `label` is coming back down in the input and covering the chip
0a204a2e-86df-4209-8acf-ba2bc648633c

File: packages/components/src/Form/Fields/FieldSelectMulti/FieldSelectMulti.stories.tsx [hunk 1]
```diff
@@ -69,12 +79,22 @@ export const Basic = Template.bind({})
...
   description: 'this is the description',
   detail: '5/50',
+  externalLabel: true,
   isFilterable: true,
   label: 'Label',
   options: selectOptions,
   placeholder: 'Search fruits',
 }
 
+export const FloatingLabel = Template.bind({})
```
Comment (laragoogle) [inline packages/components/src/Form/Fields/FieldSelectMulti/FieldSelectMulti.stories.tsx:89]: this example seems to have a problem, see gif on comments.

File: packages/components/src/Form/Fields/FieldSelectMulti/FieldSelectMulti.stories.tsx [hunk 2]
```diff
@@ -69,12 +79,22 @@ export const Basic = Template.bind({})
...
+FloatingLabel.args = { ...Basic.args, externalLabel: false }
+
+export const Values = Template.bind({})
+Values.args = { ...Basic.args, defaultValues: ['Apples', 'Oranges'] }
+
+export const ValuesFloatingLabel = Template.bind({})
```
Comment (laragoogle) [inline packages/components/src/Form/Fields/FieldSelectMulti/FieldSelectMulti.stories.tsx:95]: this one seems to have the correct behavior when values are added to the input and it is out of focus.

**Review by laragoogle — CHANGES_REQUESTED** [review]

File: packages/components/src/Form/Fields/FieldSelectMulti/FieldSelectMulti.stories.tsx (hunk 1 above)
Comment (mdodgelooker) [reply packages/components/src/Form/Fields/FieldSelectMulti/FieldSelectMulti.stories.tsx:89]: Whoops, thanks for catching that. Fix is pushed!

**Comment by laragoogle** [comment]
Hey @mdodgelooker  I'm still seeing the same issue on stories.

**Review by laragoogle — APPROVED** [review]
LGTM🚢

**Pull request merged**